core/client/app/controllers/settings/labs.js

```diff
@@ -35,6 +35,16 @@ var LabsController = Ember.Controller.extend(Ember.Evented, {
         return this.get('controllers.feature.codeInjectionUI') || false;
     }),
 
+    usePassProtectUI: Ember.computed('controllers.feature.passProtectUI', function (key, value) {
+        // setter
+        if (arguments.length > 1) {
+            this.saveLabs('passProtectUI', value);
```

Comment:
@novaugust I did that, but that didn't seem to fix the problem... The problem is occurring in either templates/settings/pass-protect.hbs or controllers/settings/pass-protect.js.